introduction/architecture.md

```diff
@@ -111,6 +110,64 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering [](timescaledb-clustering) (PRIVATE BETA)
```

Comment:
`(PRIVATE BETA)` should come before the anchor link

Reply:
Done